recipes/default.rb

```diff
@@ -73,33 +83,27 @@
   mode 0755
   action :create_if_missing
   notifies :run, 'bash[run anaconda installer]', :delayed
-end
-
-template installer_config_path do
-  source "#{installer_config}.erb"
-  user node['anaconda']['owner']
-  group node['anaconda']['group']
-  variables({
-    :version => version,
-    :flavor => flavor,
-    :anaconda_install_dir => anaconda_install_dir,
-    :accept_license => node['anaconda']['accept_license'],
-    :add_to_shell_path => 'no',
-  })
+  only_if {license_accepted}
```

Comment:
`only_if { license_accepted }`

Reply:
I'll do all corrections, but I am on vacation until September 18. I'll send a new commit at this time.